flink-ai-flow/ai_flow_plugins/scheduler_plugins/airflow/airflow_scheduler.py

```diff
@@ -340,12 +315,14 @@ def list_jobs(self, execution_id: Text) -> List[JobInfo]:
                 result = []
                 project_name, workflow_name = self.dag_id_to_namespace_workflow(dag_run.dag_id)
                 for task in task_list:
-                    job = JobInfo(job_name=task.task_id,
-                                  state=self.airflow_state_to_state(task.state),
-                                  workflow_execution
-                                  =WorkflowExecutionInfo(workflow_info=WorkflowInfo(namespace=project_name,
-                                                                                    workflow_name=workflow_name),
-                                                         execution_id=dag_run.run_id,
-                                                         state=self.airflow_state_to_state(dag_run.state)))
+                    job = JobExecutionInfo(job_name=task.task_id,
```

Comment:
The `job_execution_id` of the `JobExecutionInfo` doesn't set the value with task.try_number.